rep-2000.rst

```diff
@@ -179,6 +179,9 @@ Crystal Clemmys (December 2018 - December 2019)
 
 Support Tiers:
 
+Platforms are defined as a combination of the OS, the Architecture, and the RMW implementation.
+By default the tiers are expected to apply upto the desktop variant in a rosdistro, unless otherwise specified.
```

Comment:
It'd be great to list what is part of this `desktop variant`. Or otherwise, how to use a tool `rosinstall_generator` (?) to produce the list.

Reply:
The content of the variant is specified in a REP. Referencing that one here should be best.